oak_sev_guest/src/ghcb.rs

```diff
@@ -636,6 +624,18 @@ where
         Ok(())
     }
 
+    /// Simple wrapper for operations that rely on the GHCB.
+    fn call<F>(&mut self, func: F) -> Result<(), &'static str>
+    where
+        F: FnOnce(&mut Ghcb),
+    {
+        let ghcb = self.ghcb.as_mut();
+        ghcb.reset();
```

Comment:
Technically I don't think it is always needed to do this. I guess the overhead of zeroing out a 4KiB page is not very large compared the the VMGEXIT call, but this might make the serial port a bit slower. If it turns out to have a measurable performance impact we could perhaps pass in a bool to say whether it must be reset or not.